charts/cert-manager/v0.9.1/app-readme.md

```diff
@@ -27,3 +27,8 @@ spec:
     - myingress.com
     secretName: myingress-cert # < cert-manager will store the created certificate in this secret.
 ```
+
+#### Upgrade Notes:
+Upgrading cert-manger from previous version v0.5.2 to v0.9.1 is only supported with app deployed to the kube-system namespace.
```

Comment:
Upgrading to cert-manager from any version < v0.9.1 is *only* supported if the previous cert-manager app was deployed in the `kube-system` namespace.